oidc-authentication.md

```diff
@@ -63,6 +63,183 @@
+access mode in the case of [WAC]). Alternatively, the [entity] in control of the
+[resource server] (such as a [Solid pod] or storage space) in which the resource
+resides.
+
+Similar to the OAuth 2 and
+[UMA 2](https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html)
+concept of "resource owner", the resource controller MAY be an [end-user]
+(natural person) or MAY be a non-human entity treated as a person for limited
+legal purposes (legal person), such as a corporation.
+
+Note the use of "controller" as opposed to "owner"; this specification (and
+Solid in general), has no notion of owner.
+
+A given resource can have zero, one, or multiple resource controllers.
+
+##### protected resource
+A resource to which access is constrained by an appropriate authorization
+mechanism (such as Web Access Control [WAC] rules or Authorization Capabilities
+[zCAP]).
+
+##### client
+An application, service, bot, or other software entity interacting with a
+protected resource hosted by a [resource server] on behalf of a [requesting
+party]. The term "client" does not imply any particular implementation
+characteristics (such as whether the application executes on a server, a
+desktop, inside a web browser, or on other devices).
+
+The client is an [entity] in its own right, possessing an [identifier], and in
+some cases, a corresponding [controller document] containing client metadata.
+
+##### requesting party
+An [entity] (natural or legal person) that uses a [client] to seek access to a
+[protected resource]. The requesting party may or may not be the same party as
+the [resource controller].
+
+##### Solid pod (personal online datastore)
+A server compliant with the [Solid Specification]. For the purposes of
+[authentication], a pod acts as a peer node that acts according to one
+or more roles depending on the situation: as a [resource server] (RS), a
+[client]/[relying party], or an OpenID Identity Provider (OP).
+
+Because Solid uses an identity-based access control system ([WAC]), the access
+control process requires a combination of authenticated user and authenticated
+client. As a result, all Solid [resource servers] are also [relying parties].
+
+##### resource server (RS)
+As defined by [OAuth2]:
+
+> The server hosting the protected resources, capable of accepting and
+> responding to protected resource requests
+
+##### protected resource request
+A request by a [client] to a [protected resource], carrying [authentication] or
+authorization [credentials] such as an OAuth 2 Access Token, a Verifiable
+Credential, or Proof of Possession token.
+
+##### relying party (RP)
+As defined by [OIDC]:
+
+> OAuth 2.0 Client application requiring End-User Authentication and Claims
+> from an OpenID Provider.
+
+The [client] needs [authentication] related [claims]/[credentials] for UI reasons
+(to display a user name and icon to indicate that they’re logged in, for
+example), but also in its capacity as a [requesting party] to present those
```

Comment:
given UMA's definition of *requesting party*

> A natural or legal person that uses a client to seek access to a protected resource. The requesting party may or may not be the same party as the resource owner.

It stays distinct from the client, so client itself doesn't act as requesting party, it just acts as client and OIDC Relying Party.

Reply:
Thanks @elf-pavlik - Changed it so it doesn't say it's a requesting party.